src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/WithReferencesFindUsagesContext.cs

```diff
@@ -117,6 +117,16 @@ protected override Task OnReferenceFoundWorkerAsync(SourceReferenceItem referenc
                     addToEntriesWhenNotGroupingByDefinition: true);
             }
 
+            protected override Task OnExternalReferenceFoundWorkerAsync(ExternalReferenceItem reference)
+            {
+                // Normal references go into both sets of entries.
```

Comment:
What does this comment mean? It's not a "normal" use...?

Reply:
copypasta.  will fix.